### DISCUSSION TITLE
Finding internal parameters

### DISCUSSION THREAD

**Discussion by harish2297** [pybamm-team/PyBaMM#1642] [Q&A] [answered]
How can I find out the internal parameters like internal resistance, electrolytic / diffusion resistance and capacitance of a cell ?

**Comment by valentinsulzer** [comment, marked as answer]
That is the million-dollar question. In general it's very hard and probably the biggest weakness of the kind of models we use in pybamm. I suggest you start by reading https://iopscience.iop.org/article/10.1149/1945-7111/ab9050/meta
